src/test/java/POM/pages/BasePage.java

```diff
@@ -43,5 +47,8 @@ public AllSongsPage clickOnAllSongs(){
         return new AllSongsPage(driver);
     }
 
-
+    public CurrentQueuePage navigateToCurrentQueue() {
+        this.currentQueueMenuItem.click();
+        return new CurrentQueuePage(driver);
+    }
```

Comment:
Base Page \ Base Class — Implements the necessary methods for work with webdriver.

We will store the web elements present on each page in the corresponding java class.

The method `getUserAvatar ()` must be stored in the `class HomePage` because the avatar is visible on the web site -home page

For example, all these methods should be in the `class HomePage`



![image](https://user-images.githubusercontent.com/104761482/200763587-c43864c1-b073-495c-a6df-badc1ccc12e5.png)


```
public WebElement getUserAvatar(){
        return driver.findElement(avatarLocator);
    }

    public boolean isUserAvatarDisplayed(){
        return driver.findElement(avatarLocator).isDisplayed();
    }

    public boolean isSongPlaying() {
        WebElement soundBarVisualizer = driver.findElement(soundBarPlayLocator);
        return soundBarVisualizer.isDisplayed();
    }

    public AllSongsPage clickOnAllSongs(){
        driver.findElement(allSongsMenuItemLocator).click();
        return new AllSongsPage(driver);
    }

    public CurrentQueuePage navigateToCurrentQueue() {
        this.currentQueueMenuItem.click();
        return new CurrentQueuePage(driver);
    }
```